controllers/chat/sendSignedMessage.js

```diff
@@ -90,7 +90,7 @@ const sendSignedMesage = async (req, res) => {
     let channelMember = [];
     let senderInfo = [];
     // Prevent if user try to chat with blocked user
-    let channelType = createdChannel?.channel?.channel_type;
+    let channelType = createdChannel?.channel?.channel_type || channelType || 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a potential issue. If `createdChannel?.channel?.channel_type` is undefined, the variable `channelType` will be assigned to itself, which is also undefined at this point, before defaulting to 0. This could lead to unexpected behavior. Instead, you should directly assign the default value of 0 if `createdChannel?.channel?.channel_type` is undefined.
```diff
- let channelType = createdChannel?.channel?.channel_type || channelType || 0;
+ let channelType = createdChannel?.channel?.channel_type || 0;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->